### PR TITLE
ROC-3370 Update for service.gov.uk hostname

### PR DESCRIPTION
**Pull request by timja-kainos**
To be merged on monday

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -41,6 +41,7 @@ module "citizen-frontend" {
   is_frontend  = true
   subscription = "${var.subscription}"
   additional_host_name = "${var.external_host_name}"
+  https_only = "true"
```
Comment (geek-soft) [inline infrastructure/main.tf]: Would `https_only = true` work? (boolean instead of string)

Comment (timja-kainos) [reply infrastructure/main.tf]: nope. module is inconsistent github.com/hmcts/moj-module-webapp README

Comment (geek-soft) [reply infrastructure/main.tf]: Thanks for the link

Changing subject, I just realised we removed HPKP on CNP, something we need to enable in public BETA. Not necessarily first days.

Comment (geek-soft) [reply infrastructure/main.tf]: Yeah lets see whether Google sticks to their plan. I think so far only they made that move.

**Pull request merged**